### PR TITLE
No range

### LINKED ISSUE #44: Improve axis behavior when all values are equal
See https://github.com/RhoInc/safety-outlier-explorer/issues/48 for an example

### PR DESCRIPTION
**Pull request by samussiah**
# Issues
Closes #44

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "test-miscellaneous": "mocha --timeout 5000 --require babel-register --recursive ./test/miscellaneous/*.js",
     "test-multiply": "mocha --timeout 5000 --require babel-register --recursive ./test/multiply/*.js",
     "test-table": "mocha --timeout 5000 --require babel-register --recursive ./test/table/*.js",
+    "test-test": "mocha --timeout 5000 --require babel-register --recursive ./test/chart/setDomain.js",
```
Comment (jwildfire) [inline package.json]: meta-test!

File: src/chart/draw/consolidateData.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import naturalSorter from '../../dataOps/naturalSorter';
```
Comment (jwildfire) [inline src/chart/draw/consolidateData.js]: Assuming just the relative paths changed here ?

File: test/chart/setDomain.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import jsdom from 'jsdom';
```
Comment (jwildfire) [inline test/chart/setDomain.js]: tests look good!

File: src/chart/draw/consolidateData.js (hunk 2 above)
Comment (jwildfire) [reply src/chart/draw/consolidateData.js]: nm - 0 domain axis fix is here too ...

Comment (jwildfire) [reply src/chart/draw/consolidateData.js]: and other changes  ... :)

File: src/chart/draw/consolidateData/setDomain.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
+import naturalSorter from '../../../dataOps/naturalSorter';
```
Comment (jwildfire) [inline src/chart/draw/consolidateData/setDomain.js]: Would be good to add some comments to this code to make it easier to follow.

File: test/chart/setDomain.js (hunk 3 above)
Comment (jwildfire) [reply test/chart/setDomain.js]: I guess we could take it a step further and test all the various use cases in the huge if/then/else from setDomain(). Maybe make an issue for later?

File: src/chart/draw/consolidateData/setDomain.js (hunk 4 above)
Comment (jwildfire) [reply src/chart/draw/consolidateData/setDomain.js]: This is very nice. ConsolidateData is *a lot* easier to follow now. Well done. 💯

File: test/chart/setDomain.js (hunk 3 above)
Comment (samussiah) [reply test/chart/setDomain.js]: Yeah, the domain algorithm probably warrants some more comprehensive testing: https://github.com/RhoInc/Webcharts/issues/196

File: src/chart/draw/consolidateData/setDomain.js (hunk 4 above)
Comment (samussiah) [reply src/chart/draw/consolidateData/setDomain.js]: True!  I'll add 'em in `falsey-zeroes`.

**Comment by danedexF5** [comment]
Made the value of the STRESN column 1 for every subjID. Can confirm Y axis range is not zero.

**Comment by samussiah** [comment]
@danedexF5 , would you throw a green check mark on this please?

**Comment by danedexF5** [comment]
@samussiah I think @emmorris needs to give me the request for approval

**Comment by samussiah** [comment]
tagged you @danedexF5

**Pull request merged**